src/tests/fixtures/projects/dummy-project.yaml

```diff
@@ -609,6 +609,12 @@ project-with-rds-encryption:
             storage: 5
             encryption: arn:aws:kms:us-east-1:1234:key/12345678-1234-1234-1234-123456789012
 
+project-with-rds-major-version-upgrade:
```

Comment:
I think this needs to be added to https://github.com/elifesciences/builder/blob/master/src/tests/test_buildercore_project.py#L6

Reply:
yup, already done, just working on that other test with the adhoc alt-config definition